client/lib/media/utils/get-allowed-file-types-for-site.js

```diff
@@ -9,5 +9,5 @@ export function getAllowedFileTypesForSite( site ) {
 		return [];
 	}
 
-	return site.options.allowed_file_types;
+	return site.options.allowed_file_types ?? [];
```

Comment:
What do you think about simplifying the whole function body?

```js
return site?.options.allowed_file_types ?? []
```

Reply:
Neat idea, shortened in https://github.com/Automattic/wp-calypso/pull/78982/commits/20e23ee3aea2f3fde268daf3864c727abf6eb819